src/ssz/type/bit_vector.zig

```diff
@@ -203,3 +207,18 @@ test "BitVectorType - sanity" {
     _ = Bits.serializeIntoBytes(&b, &b_buf);
     try Bits.deserializeFromBytes(&b_buf, &b);
 }
+
+test "clone" {
+    const allocator = std.testing.allocator;
+
+    const length = 44;
+    const Bits = BitVectorType(length);
+    var b: Bits.Type = Bits.default_value;
+    try b.set(0, true);
+    try b.set(length - 1, true);
+
+    var cloned: Bits.Type = undefined;
+    try Bits.clone(allocator, &b, &cloned);
+    try std.testing.expect(&b != &cloned);
+    try std.testing.expect(std.mem.eql(u8, b.data[0..], cloned.data[0..]));
```

Comment:
Need to compare the bit_len too.

Reply:
I think you meant `bit_list` here? Fixed in [82500e7](https://github.com/ChainSafe/ssz-z/pull/51/commits/82500e79ca3e15f1e5c21576272be66bf888a9be)